src/db/file.rs

```diff
@@ -195,25 +191,7 @@ pub fn add_path_into_database<P: AsRef<Path>>(conn: &Connection,
             #[cfg(not(windows))]
             let bucket_path = bucket_path.into_os_string().into_string().unwrap();
 
-            let mime = {
-                let mime = cookie.buffer(&content)?;
-                // css's are causing some problem in browsers
-                // magic will return text/plain for css file types
-                // convert them to text/css
-                // do the same for javascript files
-                if mime == "text/plain" {
-                    let e = file_path.extension().unwrap_or_default();
-                    if e == "css" {
-                        "text/css".to_owned()
-                    } else if e == "js" {
-                        "application/javascript".to_owned()
-                    } else {
-                        mime.to_owned()
-                    }
-                } else {
-                    mime.to_owned()
-                }
-            };
+            let mime = detect_mime(&content, &file_path)?;
```

Comment:
This loads a new cookie for every item in the loop, which I imagine could have a performance inpact. It might be better to have `detect_mime` take the cookie as a parameter on linux.